### PR TITLE
Create scripts to deploy etcd with tls and StorageOS with tls

### PR DESCRIPTION
**Pull request by avestuk**
Add etcd ns to cluster role binding script args
Add cfssl binaries, update deploy etcd script

### CODE REVIEW

**Comment by avestuk** [comment]
Updated to default to StorageOS operator installation.

File: k8s/deploy-storageos/etcd-tls/cleanup-etcd.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+ETCD_NS=etcd
+ROLE_NAME=etcd-operator
+ROLE_BINDING_NAME=etcd-operator
+ETCD_OPERATOR_ROOT="./etcd-deployment/etcd-operator"
+
+# Do not delete etcd if StorageOS is running as this will destroy the StorageOS
+# cluster
+if kubectl -n storageos get pods 2>/dev/null | grep -q "STATUS"; then
+    printf "\e[31m Pods are running in the StorageOS namespace. Removing etcd resources will break the cluster so run ./storageos-deployment/cleanup.sh first\e[0m"
```
Comment (Arau) [inline k8s/deploy-storageos/etcd-tls/cleanup-etcd.sh]: The storageos-deployment/cleanup.sh doesn't exist.

**Review by Arau — COMMENTED** [review]
This is a really good piece of work.

File: k8s/deploy-storageos/etcd-tls/cleanup-etcd.sh (hunk 1 above)
Comment (avestuk) [reply k8s/deploy-storageos/etcd-tls/cleanup-etcd.sh]: Fixed path and made the error message more helpful. Now it outlines what to do for operator and manifest installations.

**Pull request merged**